base/file.jl

```diff
@@ -737,7 +737,7 @@ julia> readdir(abspath("base"), join=true)
  "/home/JuliaUser/dev/julia/base/weakkeydict.jl"
 ```
 """
-function readdir(dir::AbstractString=pwd(); join::Bool=false)
+function readdir(dir::AbstractString; join::Bool=false)
```

Comment:
I guess it's not worth it to update the docstring?

Reply:
Given that the only situation where it makes a noticeable difference is when the current directory is deleted, I think not. The doc gives a 99.9% accurate explanation of the way the function behaves unless something really weird is going on.